### DISCUSSION TITLE
Plot tree

### DISCUSSION THREAD

**Discussion by frankwwu** [pycaret/pycaret#1366] [Q&A]
Does PyCaret provide a feature similar to sklearn.tree.plot_tree? If not, is there a way to print out the decision tree from the model?

**Comment by Yard1** [comment]
Yes, `plot_model(model, plot="tree")`

**Reply by frankwwu** [reply]
I use a 'lightgbm' regression model, and it got a type error: "Decision Tree plot is only available for scikit-learn Decision Trees and Forests, Ensemble models using those or Stacked models using those as meta (final) estimators."
And the document of plot_model does not list 'tree'. [https://pycaret.org/plot-model/](https://pycaret.org/plot-model/)

**Reply by Yard1** [reply]
That documentation is outdated and will be removed. Please use https://pycaret.readthedocs.io/. LightGBM is not supported as per the exception message.

**Reply by ArtificialZeng** [reply]
> That documentation is outdated and will be removed. Please use https://pycaret.readthedocs.io/. LightGBM is not supported as per the exception message.

How to save to txt format when it's done training lightgbm. I want to know every tree node details.
